zeq/event.h

```diff
@@ -55,11 +66,22 @@ class Event : public boost::noncopyable
     ZEQ_API flatbuffers::Parser& getParser();
     ZEQ_API const flatbuffers::Parser& getParser() const;
 
+#ifdef BOOST_NO_CXX11_RVALUE_REFERENCES
+    Event& operator=( Event& rhs )
+    {
+        _impl = rhs._impl;
+        const_cast<Event &>(rhs)._impl = 0;
```

Comment:
I'd rather leave the const in both the constructor and the assignment operator. As mentioned by Juan, I also believe this is the correct declaration. No one is really happy with the const_cast, but we can admit this is a temporary solution for compilers that do not support C++11.